scripts/pre_commit_linter.py

```diff
@@ -2745,11 +2634,13 @@ def perform_all_lint_checks(self):
 
         common_messages = super(
             JsTsLintChecksManager, self).perform_all_lint_checks()
+        arg1 = (self.js_filepaths, _STDOUT_LIST, self.process_manager)
+        arg2 = (self.all_filepaths, _STDOUT_LIST, self.process_manager)
```

Comment:
I'm still not sure why we have to pass this as an argument as it's a part of the same object? Am I missing something?

Reply:
The same reason as above